jobs/JGDAS_GLOBAL_OCEAN_ANALYSIS_ECEN

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+source "${HOMEgfs}/ush/preamble.sh"
+export WIPE_DATA="NO"
+export DATA="${DATAROOT}/${RUN}ocnanal_${cyc}"
+source "${HOMEgfs}/ush/jjob_header.sh" -e "ocnanalecen" -c "base ocnanalecen"
+
+
+##############################################
+# Set variables used in the script
+##############################################
+
+##############################################
+# Begin JOB SPECIFIC work
+##############################################
+
+export PYTHONPATH=${HOMEgfs}/sorc/gdas.cd/ush:${PYTHONPATH}
```

Comment:
This does not belong in the j-job.  It should be in the rocoto job.  See example for other JEDI tasks

Reply:
Yes, this should mirror the aero jobs for example

Reply:
moved